<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QWidget" name="gridWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="cam_grid">
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Control Box</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="joystick_checkbox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Joystick </string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Time</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="time_spinbox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>60</number>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QCheckBox" name="start_button">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="2">
        <widget class="QGraphicsView" name="game1_cam">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>5</horstretch>
           <verstretch>5</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Camera</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QCheckBox" name="Camera_Checkbox">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Camera</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Score</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="score_spinbox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <number>99</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="Input_widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>5</horstretch>
           <verstretch>5</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="7" column="0">
           <widget class="QPushButton" name="angle_225">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>225°</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QPushButton" name="angle_0">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="angle_270">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>270°</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="3">
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Magnetic Field Angle</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="F_minus">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="B_minus">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Gradient Field Strength</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="F_add">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QPushButton" name="angle_315">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>315°</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Magnetic Field Strength</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="angle_90">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>90°</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="B_add">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="angle_45">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>45°</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="B_spinbox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="tabletTracking">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::WheelFocus</enum>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="accelerated">
             <bool>false</bool>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <double>24.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="angle_135">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>135°</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="angle_180">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>180°</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QDoubleSpinBox" name="theta_spinbox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="maximum">
             <double>359.990000000000009</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="F_spinbox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
            <property name="tabletTracking">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::WheelFocus</enum>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="wrapping">
             <bool>false</bool>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="accelerated">
             <bool>false</bool>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>B_add</sender>
   <signal>clicked()</signal>
   <receiver>B_spinbox</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>B_minus</sender>
   <signal>clicked()</signal>
   <receiver>B_spinbox</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>F_add</sender>
   <signal>clicked()</signal>
   <receiver>F_spinbox</receiver>
   <slot>stepUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>F_minus</sender>
   <signal>clicked()</signal>
   <receiver>F_spinbox</receiver>
   <slot>stepDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>joystick_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>angle_225</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>joystick_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>angle_180</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>558</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>joystick_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>angle_135</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>joystick_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>angle_270</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>joystick_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>angle_315</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>joystick_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>angle_0</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>joystick_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>angle_90</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>joystick_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>angle_45</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>joystick_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>F_minus</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>joystick_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>F_add</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>joystick_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>B_minus</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>joystick_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>B_add</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
